Content.Shared/Store/Components/StoreComponent.cs

```diff
@@ -87,6 +87,12 @@ public sealed partial class StoreComponent : Component
     [DataField]
     public EntityUid? StartingMap;
 
+    /// <summary>
+    /// Whether the store should have access to all listings
+    /// </summary>
+    [DataField]
+    public bool ShowAllListings = false;
+
```

Comment:
Might be better to just make listings nullable and default to all instead of having a bool that invalidates another entire field?